### PR TITLE
etcd-tester: log more for compact errors

### PR DESCRIPTION
**Pull request by gyuho**
To help debug https://github.com/coreos/etcd/issues/5440.

### CODE REVIEW

File: tools/functional-tester/etcd-tester/cluster.go [hunk 1]
```diff
@@ -329,21 +329,26 @@ func (c *cluster) compactKV(rev int64, timeout time.Duration) (err error) {
 	for i, u := range c.GRPCURLs {
 		conn, derr := grpc.Dial(u, grpc.WithInsecure(), grpc.WithTimeout(5*time.Second))
 		if derr != nil {
+			plog.Printf("#%d grpc.Dial error %v (endpoint %s)", i, derr, u)
```
Comment (xiang90) [inline tools/functional-tester/etcd-tester/cluster.go]: compact kv: dial error

File: tools/functional-tester/etcd-tester/cluster.go [hunk 2]
```diff
@@ -329,21 +329,26 @@ func (c *cluster) compactKV(rev int64, timeout time.Duration) (err error) {
...
 			err = derr
 			continue
 		}
 		kvc := pb.NewKVClient(conn)
 		ctx, cancel := context.WithTimeout(context.Background(), timeout)
+		plog.Printf("#%d compacting (endpoint %s)", i, u)
```
Comment (xiang90) [inline tools/functional-tester/etcd-tester/cluster.go]: add prefix.

File: tools/functional-tester/etcd-tester/cluster.go [hunk 3]
```diff
@@ -329,21 +329,26 @@ func (c *cluster) compactKV(rev int64, timeout time.Duration) (err error) {
...
 		_, cerr := kvc.Compact(ctx, &pb.CompactionRequest{Revision: rev, Physical: true})
 		cancel()
 		conn.Close()
 		if cerr != nil {
 			if strings.Contains(cerr.Error(), "required revision has been compacted") && i > 0 {
 				plog.Printf("%s is already compacted with %d (%v)", u, rev, cerr)
 			} else {
+				plog.Printf("#%d compact error %v (endpoint %s)", i, cerr, u)
```
Comment (xiang90) [inline tools/functional-tester/etcd-tester/cluster.go]: add prefix.

File: tools/functional-tester/etcd-tester/cluster.go [hunk 4]
```diff
@@ -329,21 +329,26 @@ func (c *cluster) compactKV(rev int64, timeout time.Duration) (err error) {
...
 				err = cerr
+				continue
 			}
 		}
+		plog.Printf("#%d compacted (endpoint %s)", i, u)
```
Comment (xiang90) [inline tools/functional-tester/etcd-tester/cluster.go]: move this to somewhere else? then we can remove the continue. do not want to change code logic for simple logging.

Comment (gyuho) [inline tools/functional-tester/etcd-tester/cluster.go]: ok will fix. thanks.

**Comment by gyuho** [comment]
/cc @xiang90 PTAL.

Thanks!

**Comment by xiang90** [comment]
lgtm

**Pull request merged**